packages/lesswrong/server/callbacks/tagRelsCallbacks.ts

```diff
@@ -0,0 +1,13 @@
+import { getCollectionHooks } from '../mutationCallbacks';
+import { Tags } from '../../lib/collections/tags/collection';
+
+getCollectionHooks("TagRels").createBefore.add(async (tagrel: DbTagRel) => {
+  const tag = await Tags.findOne({_id: tagrel.tagId});
+  if (!tag) {
+    throw new Error("Tag not found");
+  }
+  if (tag.canVoteOnRels) {
+    tagrel.canVote = tag.canVoteOnRels;
+  }
+  return tagrel;
+});
```

Comment:
The UX is great, thanks for building this! :)

(I noticed that as a normal user, I could add the "Community" tag, even though I couldn't change my vote afterwards. I assume this is intentional, though I found it slightly confusing.)

I'm worried that this logic means that **when** the tag was assigned to the post affects the voting permissions. I think it would be clearer if you could just change the settings for a tag across the board. I also couldn't think of any case when we would want to limit voting on a post or comment to just admins. So I would probably do something simpler, like having a list of topic ids in the config that could only be removed by admins, for example. I guess I'm worried that this is more abstraction than is useful.